fe1-web/model/objects/Channel.ts

```diff
@@ -11,10 +12,31 @@ export function channelFromIds(...args: Hash[]) : Channel {
       .join('/')}`;
 }
 
+/**
+ * Returns the social channel of the current user.
+ *
+ * @param laoIdHash - The hash containing the laoID of the currently opened LAO
+ */
+export function userSocialChannel(laoIdHash: Hash): Channel {
+  const userPublicKey = KeyPairStore.getPublicKey().valueOf();
+  return `${ROOT_CHANNEL}/${laoIdHash.valueOf()}/social/${userPublicKey}`;
+}
+
+/**
+ * Returns the general channel of chirps.
+ *
+ * @param laoIdHash - The hash containing the laoID of the currently opened LAO
+ */
+export function generalChirpsChannel(laoIdHash: Hash): Channel {
+  return `${ROOT_CHANNEL}/${laoIdHash.valueOf()}/social/chirps`;
+}
+
 /** Returns the last part of the channel which is usually an event id
  * Example:
- * Input: /root/laoid/electionid
- * Output: electionId
+ * Input: /root/laoID/electionID
+ * Output: electionID
+ *
+ * @param channel - The channel we want the last port
```

Comment:
```suggestion
 * @param channel - The channel whose last component we want to obtain
```

By the way, do we still need this? It seems like a very easy function to misuse

Reply:
It seems that it is used in Election.ts, and might be useful elsewhere if we want to retrieve an id. I think I'll just rename it because its name is misleading, I first thought it would return the last used channel.